### DISCUSSION TITLE
请问可以从歌词界面进入到评论区吗

### DISCUSSION THREAD

**Discussion by xxtlalala** [amll-dev/applemusic-like-lyrics#221] [Q&A] [answered]
我发现现在看不到评论区在哪里了，请问从歌词界面可以进入评论区吗，应该怎么操作。

**Comment by xxtlalala** [comment]
sorry，我搞明白了，刚开始没注意到

**Reply by LBWnb617** [reply]
怎么解决的，我还不知道怎么进去评论区

**Reply by opacscdbph44** [reply]
求助，如何从歌词节界面进入评论区

**Comment by Linho1219** [comment, marked as answer]
AMLL 歌词界面不支持进入评论区。在主界面按住 Shift 点击播放器专辑封面可唤出原生播放器，其中有评论区。

AMLL bncm 插件已停止维护，最终版本存档于 <https://github.com/amll-dev/applemusic-like-lyrics-proto>。

现关闭此讨论，敬请谅解。
